docker/.env

```diff
@@ -10,7 +11,6 @@ BAG3D_FILESTORE=/data/volume
 BAG3D_RELEASE_VERSION=develop
 BAG3D_TEST_DATA=/data/volume
 BAG3D_FLOORS_ESTIMATION_MODEL=/data/volume/model/pipeline_model1_gbr_untuned.joblib
-BAG3D_EXPORT_DIR=/data/volume/reconstruction_input/3DBAG/export_test_version
```

Comment:
removed because it was substituted by BAG3D_FILESTORE.
However, I don't see what is the difference between BAG3D_FILESTORE and BAG3D_TEST_DATA.